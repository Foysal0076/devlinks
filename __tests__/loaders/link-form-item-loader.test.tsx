import { render } from '@testing-library/react'
import LinkFormItemLoader from '@/components/loaders/link-form-item-loader'

test('renders LinkFormItemLoader correctly', () => {
  const { asFragment } = render(<LinkFormItemLoader />)
  expect(asFragment()).toMatchSnapshot()
})
