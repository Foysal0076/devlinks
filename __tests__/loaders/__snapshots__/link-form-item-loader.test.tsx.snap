// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LinkFormItemLoader correctly 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col rounded-lg bg-neutral-10 p-5 pb-6 dark:bg-surface-200/40"
  >
    <div
      class="mb-4 flex items-center justify-between"
    >
      <div
        class="flex items-center"
      >
        <div
          class="bg-loader h-6 w-6 animate-pulse rounded"
        />
        <div
          class="bg-loader ml-2 h-6 w-24 animate-pulse rounded"
        />
      </div>
      <div
        class="bg-loader mb-1.5 h-4 w-20 animate-pulse rounded"
      />
    </div>
    <div
      class="flex flex-col gap-8"
    >
      <div
        class="bg-loader h-10 w-full animate-pulse rounded"
      />
      <div
        class="bg-loader h-10 w-full animate-pulse rounded"
      />
    </div>
  </div>
</DocumentFragment>
`;
